json/src/lib.rs

```diff
@@ -1580,13 +1583,13 @@ pub struct GetBlockTemplateResult {
     /// List of transactions included in the template block
     pub transactions: Vec<GetBlockTemplateResultTransaction>,
     /// The signet challenge. Only set if mining on a signet, otherwise empty
-    #[serde(default, with = "bitcoin::blockdata::script::Script")]
-    pub signet_challenge: bitcoin::blockdata::script::Script,
+    #[serde(default, with = "bitcoin::blockdata::script::ScriptBuf")]
```

Comment:
Everything in `blockdata` is re-exported at the crate root so no need for `::blockdata` here or anywhere in this repo.

Reason, incase its not widely known, is that we are trying to separate the API from the directory structure and "blockdata" is an code/module organisation bucket not an API thing.

Reply:
sure, `::blockdata` removed